src/source-dpdk.c

```diff
@@ -507,10 +507,10 @@ static TmEcode ReceiveDPDKThreadInit(ThreadVars *tv, const void *initdata, void
         // some PMDs requires additional actions only after the device has started
         DevicePostStartPMDSpecificActions(ptv, dev_info.driver_name);
 
-        uint16_t inconsist_numa_cnt = SC_ATOMIC_GET(dpdk_config->inconsitent_numa_cnt);
-        if (inconsist_numa_cnt > 0) {
+        uint16_t inconsistent_numa_cnt = SC_ATOMIC_GET(dpdk_config->inconsitent_numa_cnt);
+        if (inconsistent_numa_cnt > 0) {
```

Comment:
I can't find the `inconsitent_numa_cnt` token in dpdk: https://github.com/search?q=repo%3ADPDK%2Fdpdk%20inconsitent&type=code

(I sent them PRs a while ago...)

But I left it unchanged under the assumption that it's an upstream element.

Having now searched more carefully, I think I was too cautious here.